IN-global_conf.json

```diff
@@ -1,78 +1,79 @@
 {
 	"SX1301_conf": {
 		"lorawan_public": true,
-		"clksrc": 1,
+		"clksrc": 0,
```

Comment:
OK, do you want me to make the change and repost? (I haven't any way to test the non-Conduit use.)